test/localisationsystemSERVER/carclientserver.py

```diff
@@ -32,8 +32,8 @@
 import socket
 import time
 
-from loc_sys_server.server.utils import load_private_key, sign_data
-from loc_sys_server.server.complexDealer import ComplexEncoder
+from utils import load_private_key, sign_data
+from complexDealer import ComplexEncoder
```

Comment:
Since the directory structure was changed, the imports also had to be refactored, otherwise, a "ModuleNotFound" error was thrown.